src/components/missions/MissionItem.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { joinMission, leaveMission } from '../../redux/missions/missions';
+import './MissionItem.css';
+
+function MissionItem({ mission }) {
+  const {
+    mission_name, mission_id, description, reserved,
+  } = mission;
+  const dispatch = useDispatch();
+  const handleJoin = () => {
+    dispatch(joinMission(mission_id));
+  };
+
+  const handleLeave = () => {
+    dispatch(leaveMission(mission_id));
+  };
+
+  return (
+    <tr>
+      <td className="mis-name align-left">
+        <h3>{mission_name}</h3>
+      </td>
+      <td className="mis-descriptions align-left">{description}</td>
+      <td className="mis-btn1">
+        {reserved
+          ? <button type="button" className="badge-secondary">Active member</button>
+          : <button type="button" className="badge-primary">Not a member</button>}
+
+      </td>
+      <td className="mis-btn2">
+        {!reserved
+          ? (
+            <button type="button" className="btn-primary" onClick={handleJoin}>Join Mission</button>
+          )
+          : <button type="button" className="btn-secondary" onClick={handleLeave}>Leave Mission</button>}
+      </td>
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/156244653-5599d59b-03ea-4689-aeb5-e6565b48134c.png)

Therefore:

- [ ] Please, kindly use the React conditional rendering syntax, instead of using ternary operators.

## Hints

You could build if-else statements using this syntax by using the `!` (NOT) operator like this: `{!mission.reserved && (what to do if the previous thing is true)}` and still meet the requirement 💪